docs/self-managed/setup/guides/connect-to-an-oidc-provider.md

```diff
@@ -124,13 +128,20 @@ Ensure you register a new application for each component.
 :::
 
 1. Within the Entra ID admin center, [register a new application](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app) for **each** component you would like to connect.
-2. Navigate to the new application's **Overview** page, and make note of the **Client ID**.
+2. Navigate to the new application's **Overview** page, and make note of the **Client ID**. This will also be used as the audience ID.
 3. Within your new application, [configure a platform](https://learn.microsoft.com/en-gb/entra/identity-platform/quickstart-register-app#configure-platform-settings) for the appropriate component:
    - **Web**: Operate, Tasklist, Optimize, Identity
    - **Single-page application**: Modeler, Console
 4. Add your component's **Microsoft Entra ID** redirect URI, found under [Component-specific configuration](#component-specific-configuration).
-5. [Create a new client secret](https://learn.microsoft.com/en-gb/entra/identity-platform/quickstart-register-app?tabs=client-secret#add-credentials), and note the new secret's value for later use.
-6. Set the following environment variables for the component you are configuring an app for:
+   :::note
+   In Microsoft Entra ID, redirect URIs serve as an approved list of destinations. Only the URLs specified in the redirect URIs configuration will be permitted as valid redirection targets for authentication responses. This security measure ensures that tokens and authorization codes are only sent to pre-approved locations, preventing potential unauthorized access or token theft.
```

Comment:
🔧 I would suggest that this is a little too specific, the redirect URI concept is not just relevant to Entra but all auth providers

Reply:
Other way around @hamza-m-masood - this note makes it seem like redirect URIs are something entra specific but in reality they are applicale to all providers

Reply:
I'm fine with removing it.

Reply:
sorry but just to bring one more cook to the kitchen, why not rephrasing it then and make it more general? Might be still a good note for customers being new to this whole topic: 

```
Redirect URIs serve as an approved list of destinations across identity providers. Only the URLs specified in the redirect URIs configuration will be permitted as valid redirection targets for authentication responses. This security measure ensures that tokens and authorization codes are only sent to pre-approved locations, preventing potential unauthorized access or token theft.
```

Reply:
Works for me! I went ahead and used this generic version, and also added it to the generic section for completeness.